dask/dataframe/tests/test_dataframe.py

```diff
@@ -3160,6 +3166,24 @@ def test_apply():
         ddf.apply(lambda xy: xy, axis="index")
 
 
+@pytest.mark.parametrize("convert_dtype", [True, False])
```

Comment:
Do we test `convert_dtype` anywhere else? If not, it'd be good to test that no warning is emitted by default